### PR TITLE
(feat) O3-2150: Add Hebrew to openmrs-esm-patient-management

### PR DESCRIPTION
**Pull request by AlexanderMizgirev**
## Requirements

- [x] This PR has a title that briefly describes the work done including the ticket number. If there is a ticket, make sure your PR title includes a [conventional commit](https://o3-dev.docs.openmrs.org/#/getting_started/contributing?id=your-pr-title-should-indicate-the-type-of-change-it-is) label. See existing PR titles for inspiration.
- [x] My work conforms to the [**OpenMRS 3.0 Styleguide**](https://om.rs/styleguide) and [**design documentation**](https://zeroheight.com/23a080e38/p/880723-introduction).
- [x] I checked for feature overlap with [**existing widgets**](https://om.rs/directory).

## Summary
Added Hebrew translation files to openmrs-esm-patient-management

## Screenshots

*None.*

## Related Issue
https://issues.openmrs.org/browse/O3-2150

## Other

*None.*

### CODE REVIEW

File: packages/esm-appointments-app/translations/he.json [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+  "encounterType": "סוג הפגישה",
+  "endTime": "שעת סיום",
+  "errorAppoinmentMetric": "",
+  "errorMessage": "שגיאה בביטול התור",
+  "exitAndChangeStatus": "שינוי סטטוס",
+  "facility": "מוסד",
+  "filter": "סינון",
+  "filterByServiceType": "",
+  "frequency": "תדירות",
+  "futureScheduledVisits": " {count, plural, one {{count} ביקור} two {{count} ביקורים} many {{count} ביקורים} other {{count} ביקורים}} scheduled for dates in the future",
```
Comment (ibacher) [inline packages/esm-appointments-app/translations/he.json:73]: Shouldn't the "scheduled for dates in the future" part also be in Hebrew?

File: packages/esm-appointments-app/translations/he.json [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+  "priority": "עדיפות",
+  "promisedPateints": "מטופלים עם הבטחה",
+  "promisedPatients": "מטופלים עם הבטחה",
+  "provider": "ספק",
+  "providers": "ספקים",
+  "providersAvailableToday": "ספקים זמינים: {time}",
+  "queue": "תור",
+  "queueEntryError": "שגיאה בהוספת המטופל לתור",
+  "reasonForChanges": "סיבת השינויים",
+  "recentScheduledVisits": "{count, plural, one {{count} ביקור} two {{count} ביקורים} many {{count} ביקורים} other {{count} ביקורים}} scheduled for +/- 7 days",
```
Comment (ibacher) [inline packages/esm-appointments-app/translations/he.json:109]: Ditto

File: packages/esm-patient-list-app/translations/he.json [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+"searchForList": "חפש רשימה",
+"selectCohortType": "בחר סוג רשימת מטופלים",
+"sex": "מין",
+"startDate": "תאריך התחלה",
+"successAddPatientToList": "המטופל נוסף לרשימה",
+"successCreatedPatientList": "נוצרה רשימת מטופלים",
+"successCreatedPatientListDescription": "יצירת רשימת מטופלים בהצלחה",
+"successfullyAdded": "נוסף בהצלחה",
+"successUpdatePatientList": "עודכנה רשימת מטופלים",
+"successUpdatePatientListDescription": "עדכון רשימת מטופלים בהצלחה"
```
Comment (ibacher) [inline packages/esm-patient-list-app/translations/he.json:49]:
```suggestion
  "addPatientToList": "הוסף מטופל לרשימה",
  "addToList": "הוסף לרשימה",
  "cancel": "ביטול",
  "configureList": "הגדר את רשימת המטופלים שלך באמצעות השדות למטה",
  "createdOn": "נוצר בתאריך",
  "createList": "צור רשימה",
  "createNewPatientList": "צור רשימת מטופלים חדשה",
  "delete": "מחק",
  "deleted": "נמחק",
  "deletedPatientList": "רשימת מטופלים נמחקה",
  "editList": "ערוך רשימה",
  "editNameDescription": "ערוך שם/ תיאור",
  "editPatientListHeader": "ערוך רשימת מטופלים",
  "error": "שגיאה",
  "errorAddPatientToList": "המטופל לא נוסף לרשימה",
  "errorCreatePatientListDescription": "לא ניתן ליצור רשימת מטופלים",
  "errorDeleteList": "לא ניתן למחוק רשימת מטופלים",
  "errorUpdatePatientListDescription": "לא ניתן לעדכן רשימת מטופלים",
  "identifier": "מזהה",
  "items": "פריטים",
  "listDescriptionPlaceholder": "לדוגמה, מטופלים עם אסתמה שנמצאים בראשית הדרך להשתתפות במחקר אוניברסיטאי",
  "listNamePlaceholder": "לדוגמה, משתתפי מחקר פוטנציאליים",
  "name": "שם",
  "newList": "רשימה חדשה",
  "newPatientListCohortTypeLabel": "בחר את סוג הקוהורט לרשימת המטופלים החדשה",
  "newPatientListDescriptionLabel": "תאר את מטרת הרשימה במספר מילים",
  "newPatientListHeader": "רשימת מטופלים חדשה",
  "newPatientListNameLabel": "שם הרשימה",
  "noPatientListFound": "לא נמצאה רשימת מטופלים",
  "offlinePatients": "מטופלים במצב לא מקוון",
  "openPatientList": "הוסף לרשימה",
  "patientList": "רשימת מטופלים",
  "patientListAppMenuLink": "רשימות מטופלים",
  "patientLists": "רשימות מטופלים",
  "patients": "מטופלים",
  "search": "חיפוש",
  "searchForAListToAddThisPatientTo": "חפש רשימה להוספת המטופל לתוכה.",
  "searchForList": "חפש רשימה",
  "selectCohortType": "בחר סוג רשימת מטופלים",
  "sex": "מין",
  "startDate": "תאריך התחלה",
  "successAddPatientToList": "המטופל נוסף לרשימה",
  "successCreatedPatientList": "נוצרה רשימת מטופלים",
  "successCreatedPatientListDescription": "יצירת רשימת מטופלים בהצלחה",
  "successfullyAdded": "נוסף בהצלחה",
  "successUpdatePatientList": "עודכנה רשימת מטופלים",
  "successUpdatePatientListDescription": "עדכון רשימת מטופלים בהצלחה"
```

File: packages/esm-appointments-app/translations/he.json (hunk 1 above)
Comment (AlexanderMizgirev) [reply packages/esm-appointments-app/translations/he.json]: Fixed. Thank you! I also noticed that other languages have the same issue in these places.

**Review by ibacher — APPROVED** [review]

**Pull request merged**